.config-schema.json

```
@@ -430,6 +430,29 @@
                     "x-env-variable": "OPENFGA_CHECK_QUERY_CACHE_TTL"
                 }
             }
+        },
+        "dispatchThrottling": {
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "enable throttling when request's number of dispatches is high",
+                    "type": "bool",
+                    "default": "false",
+                    "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_ENABLED"
+                },
+                "timeTickerFrequency": {
+                    "description": "defines how frequent dispatch throttling will be evaluated",
+                    "type": "duration",
+                    "default": "10µs",
+                    "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_TIME_TICKER_FREQUENCY"
+                },
+                "level": {
```

IMO "level" doesn't convey what this is. Perhaps:

```suggestion
                "limit": {
```
or
```suggestion
                "throttling_threshold": {
```
or

```suggestion
                "threshold": {
```

---

Changed to threshold